README.md

```diff
@@ -1 +1,52 @@
-# ToDoList
\ No newline at end of file
+# ToDoList
```

Comment:
- [ ] You need to move all the files in the `webpack` folder to the root directory as the root directory is where the app start command should work from.